src/openface_node.py

```diff
@@ -213,15 +247,16 @@ def _detect_face_srv(self, req):
         # Try to recognize
         detections = self._update_detections_with_recognitions(detections)
 
-        rospy.logerr("This is for debugging")
-        rospy.logerr(detections)
+        #rospy.logerr("This is for debugging")
+        #rospy.logerr(detections)
```

Comment:
Put a comment in the code but basically the original version did too much logging.